src/contracts/core/DelegationManager.sol

```diff
@@ -477,6 +482,12 @@ contract DelegationManager is
                 });
             }
 
+            if (depositSharesToWithdraw[i] == shareManager.stakerDepositShares(staker, strategies[i])) {
+                //Reset DepositScalingFactor
+                DepositScalingFactor storage dsf = _depositScalingFactor[staker][strategies[i]];
+                dsf.reset();
```

Comment:
Nit: can we make these one-liners? since the method doesnt have any args i dont think there's value in splitting it into 2 LOC